packages/components/src/internal/actions.ts

```diff
@@ -2459,6 +2459,35 @@ function beginPaste(model: EditorModel, numRows: number): EditorModel {
     });
 }
 
+export function addRowsPerPivotValue(model: QueryGridModel, numPerParent: number, pivotKey: string, pivotValues: Array<string>, rowData: Map<string, any>): EditorModel {
+    let editorModel = getEditorModel(model.getId());
+    let data = model.data;
+    let dataIds = model.dataIds;
+    let totalItems = 0;
```

Comment:
I don't see anywhere that this is actually used (other then this init and the update to add to it).